### ISSUE TITLE
[Request] Adding missing DataTables libraries

### ISSUE THREAD

**Issue by marceloverdijk** [cdnjs/cdnjs#13978]
[DataTables](https://datatables.net/) and some of it [extensions](https://datatables.net/extensions/index) are already on cdnjs but some not.

It would be nice to add all DataTables extensions to cdnjs.

All extensions and the various dist packages can be found at https://github.com/DataTables?tab=repositories&q=Dist&type=&language=

**Comment by marceloverdijk** [comment]
Note that the main DataTables library in cdnjs is out of date (latest release `1.10.21` but on cdnjs `1.10.20`). See https://cdnjs.com/libraries/datatables

I don't know what cdnjs is using as source but the @DataTables seems not to use Github Releases anymore. See this discussion https://datatables.net/forums/discussion/62530/1-10-21-release-not-on-github-and-https-cdnjs-com#latest. So this might be related?

**Comment by MattIPv4** [comment]
cdnjs relies on the git tags in https://github.com/DataTables/DataTables for the main DataTables library. If that is no longer being updated, we welcome suggestions of where we can pull updates from that contain the same set of files.

As for other extensions, I believe we're pulling the current ones from NPM and I can add the rest later 👍

**Comment by MattIPv4** [comment]
That should be all the datatables.net packages that meet the cdnjs popularity requirement. Can't beleive how many there are!

**Comment by marceloverdijk** [comment]
Yes indeed there are many

**Comment by DataTables** [comment]
Yup - it feels ridiculous, but its due to the support we have for various styling libraries. It increases our package count by 6n.

I'll sync up the original DataTables/DataTables repo and get it tagged up.

**Comment by marceloverdijk** [comment]
Wow thx a lot.
